### PR TITLE
User/delete data

### PR DESCRIPTION
**Pull request by O-tega**

### CODE REVIEW

File: app/Controllers/Http/PetsController.ts [hunk 1]
```diff
@@ -34,6 +34,9 @@ export default class PetsController {
 		}
 	}
 
+	// New files in between
```
Comment (Okeibunor) [inline app/Controllers/Http/PetsController.ts:37]: please remove this comment.

File: app/Controllers/Http/PetsController.ts [hunk 2]
```diff
@@ -60,6 +63,9 @@ export default class PetsController {
 		}
 	}
 
+
+	// Creating description for git
```
Comment (Okeibunor) [inline app/Controllers/Http/PetsController.ts:67]: please remove this comment

File: app/Controllers/Http/UsersController.ts [hunk 3]
```diff
@@ -85,6 +90,7 @@ export default class AuthController {
 			const emailExists = await User.query().where('email', email).first()
 
 			if (emailExists) {
+				Logger.error({err: new Error("Email alredy aexist")}, "Email address provided already exist in the database")
```
Comment (Okeibunor) [inline app/Controllers/Http/UsersController.ts:93]: typo "aexist"

File: app/Controllers/Http/PetsController.ts (hunk 1 above)
Comment (O-tega) [reply app/Controllers/Http/PetsController.ts]: ok

File: app/Controllers/Http/UsersController.ts (hunk 3 above)
Comment (O-tega) [reply app/Controllers/Http/UsersController.ts:93]: ok boss

**Review by Okeibunor — APPROVED** [review]

**Pull request merged**